src/module.c

```diff
@@ -3042,6 +3042,23 @@ client *moduleGetReplyClient(ValkeyModuleCtx *ctx) {
     }
 }
 
+int VM_UpdateRunTimeArgs(ValkeyModuleCtx *ctx, int argc, ValkeyModuleString **argv) {
```

Comment:
Implementation looks good.

The name here is "RunTime" but in the PR title it says "Runtime". Question to native English speakers: Is it one word (runtime) or two words (run time)?

Reply:
There is one exception named RuntimeException  in Java, thus the correct function should be VM_UpdateRuntimeArgs(...) ?

Reply:
Addressed, update the function name to VM_UpdateRuntimeArgs(...)